Avoid using cmdlet aliases

f-bader opened the pull request:
'Select' is an alias of 'Select-Object'. Alias can introduce possible problems and make scripts hard to maintain.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

f-bader committed:
Avoid using cmdlet aliases

'Select' is an alias of 'Select-Object'. Alias can introduce possible problems and make scripts hard to maintain.
